xbmc/addons/AddonDatabase.cpp

```diff
@@ -1073,7 +1073,7 @@ void CAddonDatabase::GetInstallData(const AddonInfoPtr& addon)
                                         CDateTime::FromDBDateTime(m_pDS->fv(3).get_asString()),
                                         CDateTime::FromDBDateTime(m_pDS->fv(4).get_asString()),
                                         CDateTime::FromDBDateTime(m_pDS->fv(5).get_asString()),
-                                        m_pDS->fv(6).get_asString());
+                                        m_pDS->fv(7).get_asString());
```

Comment:
I think this change slipped in from another change and it's unrelated to this PR.

Reply:
changed this query in #18269 - sorry @ronie this was going in parallel

Reply:
How this is changed in #18269 is better IMO.

Reply:
This is actually in a temp commit so should be ignored.

Reply:
this was just a quick fix applied on my end (hence the WIP label) so i could properly test these changes.